packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -73,7 +73,7 @@
 }
 
 .plan-details-component {
-  @apply my-0 mx-4 p-0;
+  @apply mt-0 mx-4 mb-8 p-0 tablet:h-0;
```

Comment:
Adding `mb-8` affects all subscription create and upgrade routes in mobile. Extra margin is added below `PlanDetails` and above the new user form and/or payment containers.

I recommend removing `mb-8` and passing in a custom class with a top margin consistent with the spacing between all containers in mobile (see my other comment for details).

Additionally, what does `tablet:h-0` do? I know it was in the stylesheet but is it necessary or can it be removed altogether?

Reply:
If I'm reading+understanding https://tailwindcss.com/docs/height correctly, wouldn't `h-0` set `height: 0px;` for the tablet media query breakpoint?

Reply:
Reverted! `tablet:h-0` eliminated this from happening:
<img width="1068" alt="Screen Shot 2022-11-08 at 7 20 02 PM" src="https://user-images.githubusercontent.com/28129806/200705144-34cd5415-a2a8-4330-9e36-7d45ecc7f67b.png">